salt/states/file.py

```diff
@@ -1882,6 +1882,9 @@ def absent(name, **kwargs):
     be deleted. This will work to reverse any of the functions in the file
     state module. If a directory is supplied, it will be recursively deleted.
 
+    If only the contents of the directory need to be deleted but not the directory itself, use :mod:`file.directory
```

Comment:
Can we honor the 80 character line limit?

Reply:
I added a PR here https://github.com/saltstack/salt/pull/61495

Reply:
Thank you for correcting @twangboy, I'll make sure to follow it now it.